web-frontend/src/layouts/Placeholder/Placeholder.tsx

```diff
@@ -0,0 +1,22 @@
+import { FiList } from 'react-icons/fi';
+import Intl from '../../i18n/i18n';
+
+function Placeholder({ ...props }) {
+  const { message } = props;
+  return (
+    <div
+      className="flex flex-col justify-around items-center  
+        h-96 w-[700px] p-10 gap-10
```

Comment:
For chat you put `w-[700px]` ?
That's not work with `w-fit` ?

Reply:
![Screenshot from 2023-06-06 17-12-33](https://github.com/massalabs/thyra-plugin-wallet/assets/90157528/ff5b3537-5c8e-4320-ba78-d07bc9a798bc)

Reply:
> Weird... this centralisation should be fixed.
> 
> ... and I guess the icon is not that one neither, right!?

I have an idea to fix this problem. I will test it.